app.config.ts

```diff
@@ -1,71 +1,75 @@
-import { ExpoConfig, ConfigContext } from '@expo/config';
-import { version } from './package.json'
+import { ExpoConfig, ConfigContext } from "@expo/config"
+import { version } from "./package.json"
 
-const BUILD_NUMBER = 2;
+const BUILD_NUMBER = 2
 
 export default ({ config }: ConfigContext): ExpoConfig => ({
   ...config,
-  name: 'Chain React App 2023',
-  slug: 'ChainReactApp2023',
+  name: "Chain React App 2023",
+  slug: "ChainReactApp2023",
+  scheme: "chainreactapp",
   version,
-  orientation: 'portrait',
-  icon: './assets/images/app-icon-all.png',
+  orientation: "portrait",
+  icon: "./assets/images/app-icon-all.png",
   splash: {
-    image: './assets/images/splash-logo-all.png',
-    resizeMode: 'contain',
-    backgroundColor: '#081828'
+    image: "./assets/images/splash-logo-all.png",
+    resizeMode: "contain",
+    backgroundColor: "#081828",
   },
   updates: {
-    fallbackToCacheTimeout: 0
+    fallbackToCacheTimeout: 0,
   },
-  jsEngine: 'hermes',
-  assetBundlePatterns: ['**/*'],
+  jsEngine: "hermes",
+  assetBundlePatterns: ["**/*"],
   android: {
-    icon: './assets/images/app-icon-android-legacy.png',
-    package: 'com.chainreactapp',
+    icon: "./assets/images/app-icon-android-legacy.png",
+    package: "com.chainreactapp",
     versionCode: BUILD_NUMBER,
     adaptiveIcon: {
-      foregroundImage: './assets/images/app-icon-android-adaptive-foreground.png',
-      backgroundImage: './assets/images/app-icon-android-adaptive-background.png'
+      foregroundImage: "./assets/images/app-icon-android-adaptive-foreground.png",
+      backgroundImage: "./assets/images/app-icon-android-adaptive-background.png",
     },
     splash: {
-      image: './assets/images/splash-logo-android-universal.png',
-      resizeMode: 'contain',
-      backgroundColor: '#081828'
+      image: "./assets/images/splash-logo-android-universal.png",
+      resizeMode: "contain",
+      backgroundColor: "#081828",
     },
-    googleServicesFile: `./google-services.json`
+    googleServicesFile: `./google-services.json`,
   },
   ios: {
-    icon: './assets/images/app-icon-ios.png',
+    icon: "./assets/images/app-icon-ios.png",
     supportsTablet: true,
-    bundleIdentifier: 'infinitered.stage.ChainReactConf',
+    bundleIdentifier: "infinitered.stage.ChainReactConf",
     buildNumber: String(BUILD_NUMBER),
     splash: {
-      image: './assets/images/splash-logo-ios-mobile.png',
-      tabletImage: './assets/images/splash-logo-ios-tablet.png',
-      resizeMode: 'contain',
-      backgroundColor: '#081828'
+      image: "./assets/images/splash-logo-ios-mobile.png",
+      tabletImage: "./assets/images/splash-logo-ios-tablet.png",
+      resizeMode: "contain",
+      backgroundColor: "#081828",
+    },
+    googleServicesFile: `./GoogleService-Info.plist`,
+    infoPlist: {
```

Comment:
Probably going to have a merge conflict here due to the hidden status bar but should be an easy merge.